spec/lib/vmdb/settings_spec.rb

```diff
@@ -202,4 +214,25 @@
 
     include_examples "password handling"
   end
+
+  describe ".for_resource" do
+    let(:server) { FactoryGirl.create(:miq_server) }
+
+    it "without database changes" do
+      settings = Vmdb::Settings.for_resource(server)
+      expect(settings.api.token_ttl).to eq "10.minutes"
+    end
+
+    it "with database changes" do
+      server.settings_changes.create!(:key => "/api/token_ttl", :value => "2.minutes")
+      settings = Vmdb::Settings.for_resource(server)
+      expect(settings.api.token_ttl).to eq "2.minutes"
+    end
+
+    it "with database changes on an Array" do
+      server.settings_changes.create!(:key => "/log/collection/current/pattern", :value => ["*.log"])
```

Comment:
If I understand this correctly, these tests are testing this change only if the settings.yml originally has an array in `"/log/collection/current/pattern`.  Does it make sense to not depend on the current values in the settings.yml by injecting a desired value for it?